sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ReceiverManager.cs

```diff
@@ -67,6 +69,7 @@ internal class ReceiverManager
             {
                 try
                 {
+                    _closingTokenSource.Cancel();
```

Comment:
On further inspection, I don't think is needed. We already cancel the renew lock token in the finally clause of ProcessOneMessage.

Reply:
I suspect the underlying issue is due to the very short lock duration these tests were using.